.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,8 +6,7 @@ Resolves: python-poetry#<!-- add issue number/link here -->
 - [ ] Updated **documentation** for changed code.
 
 <!--
-**Note**: If your Pull Request introduces a new feature or changes the current behavior, it should be based
-on the `develop` branch. If it's a bug fix or only a documentation update, it should be based on the `master` branch.
+**Note**: All Pull Request should be based on the `main` branch.
```

Comment:
Better to use the plural form.

I also think that using **must** put more emphasis

```suggestion
**Note**: All Pull Requests must be based on the `main` branch.
```

Reply:
I tend to "should" but no strong feelings about that.

+1 for plural